pallets/claim/Cargo.toml

```diff
@@ -0,0 +1,45 @@
+[package]
+name = "pallet-claim"
+version = "0.1.0"
+description = "A pallet to claim airdropped tokens"
+authors.workspace = true
+edition.workspace = true
+homepage.workspace = true
+repository.workspace = true
+license = "Apache-2.0"
+
+[dependencies]
+log = "0.4.20"
```

Comment:
Maybe could be a a nice idea to start to move all crates dependency (at least the common ones) in the workspace Cargo.toml.

What do you think?